tests/unit/amo/pages/TestFeedback.js

```diff
@@ -187,15 +179,32 @@ describe(__filename, () => {
     ['report', 'other'],
     ['feedback', 'does_not_work'],
     ['feedback', 'feedback_spam'],
-  ])(`renders reason %s`, (category, reasonSlug) => {
-    const categories = getCategories(fakeI18n());
+  ])(`renders reason %s for extensions`, (category, reasonSlug) => {
```

Comment:
I think we've covered all the cases in these tests but it's difficult to tell without carefully reading them as they're all slightly different.  For example, this test checks each category one by one and renders; while `omit some categories when add-on type is not extension` is checking the contents of the categories object instead.  And then `'renders a form for a %s'` below tests for the addon types that don't have all the categories while extension type is handled above, but not explicitly.  

It's a hassle I know, but could you rework the tests so it's clearly &lt;testing extension and it does this&gt; and then each(&lt;testing this other type and it does that&gt;)

Reply:
Yep, I rewrote the tests to explicitly cover the differences between add-on types AND verify all the displayed categories.